package.json

```diff
@@ -32,7 +32,7 @@
     "fetch": "sh src/scripts/fetch.sh",
     "init:generated": "mkdirp ./generated/loaders && mkdirp ./generated/plugins ",
     "lint": "run-s lint:*",
-    "lint:links": "hyperlink -r build/index.html --canonicalroot https://webpack.js.org/ --debug | node ./src/scripts/check-links.js",
+    "lint:links": "hyperlink -r build/index.html --canonicalroot https://webpack.js.org/",
```

Comment:
:)

Reply:
I just can't recreate this damn failure :'(